packages/core/src/crypto/__tests__/JwsService.test.ts

```diff
@@ -1,28 +1,35 @@
 import type { AgentContext } from '../../agent'
 import type { Key, Wallet } from '@aries-framework/core'
 
-import { IndySdkWallet } from '../../../../indy-sdk/src'
-import { indySdk } from '../../../../indy-sdk/tests/setupIndySdkModule'
-import { getAgentConfig, getAgentContext } from '../../../tests/helpers'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@hyperledger/aries-askar-nodejs'
```

Comment:
Why is this required? Can we not just register it or let the `index.js` of that package deal with it?

Reply:
We need to import it, as we use askar here. But I can add this to the `setup.ts` file